app/views/mailers/member_mailer/access_revoked_manager_email.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render partial: "mailers/email_template",
+locals: {
+  url:
+    (
+      if @namespace.type == "Group"
+        group_members_url(@namespace)
+      else
+        namespace_project_members_url(@namespace.parent, @namespace.project)
+      end
+    )
+} %>
+
+<%= yield :header_and_greeting %>
+
+<tr>
+  <td style="padding:12px 24px;">
+    <p style="color:#334155;line-height:24px;">
+      <%= t(
```

Comment:
**nitpick (blocking):** Can you display the user that has been revoked access like below:

`John Doe (john.doe@email.com)`

In this case email address is our unique identifier for a user so this is clearer on who has been granted access.